package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "the-weather-app",
+  "version": "1.0.0",
+  "description": "Weather application",
+  "private": "true",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
- [ ] Please install the web-dev-server as running `npm start` returns an error.